public/locales/en.json

```diff
@@ -306,6 +306,7 @@
                 "line": "Selected {{count}} lines"
             },
             "lines": "{{lines}} Lines",
+            "words": "{{words}} Words",
```

Comment:
Do we handle plurals here?

Reply:
I didn't do that here as we didn't do that at many other places as well. Maybe that's a thing for a separate issue?